coderabbit.yaml

```
@@ -0,0 +1,60 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+
+tone_instructions: "constructive"
+review_instructions: |
+  Please use this structured format for all code reviews:
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)